meilisearch-http/src/index/search.rs

```diff
@@ -247,6 +252,25 @@ impl Index {
     }
 }
 
+fn insert_geo_distance(sorts: &[String], document: &mut Document) {
+    lazy_static::lazy_static! {
+        static ref GEO_REGEX: Regex =
+            Regex::new(r"_geoPoint\(\s*([[:digit:].\-]+)\s*,\s*([[:digit:].\-]+)\s*\)").unwrap();
+    };
+    if let Some(capture_group) = sorts.iter().find_map(|sort| GEO_REGEX.captures(sort)) {
+        // TODO: TAMO: milli encountered an internal error, what do we want to do?
```

Comment:
Should this stay? Can we open an issue instead? :)

Reply:
I wanted to know what @MarinPostma thought of this; I think we can take a decision right away and remove this TODO entirely.